### ISSUE TITLE
Have a designer check the accessibility of our site

### ISSUE THREAD

**Issue by svpino** [underfitted/bnomial-daily#149]
I'd like to make sure our site can be used by everyone. 

We should have somebody that knows how to objectively evaluate accessibility problems take a look at our site and offer suggestions of how to make it accessible to everyone.

Is there maybe a service that does this?

**Comment by haltakov** [comment]
Lighthouse already gives some suggestions on that, but we should probably take a deeper look.
